com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -187,24 +224,34 @@ private static Quaternion ToQuaternion(Vector3 eigenvector0, Vector3 eigenvector
 #endregion
 
 #region Export
-        public Link.Inertial ExportInertialData() 
+        public Link.Inertial ExportInertialData()
         {
-            ArticulationBody robotLink = GetComponent<ArticulationBody>();
+            var robotLink = GetComponent<ArticulationBody>();
 
             if (robotLink == null)
+            {
+                Debug.LogWarning("No data to export.");
                 return null;
+            }
 
             UpdateLinkData();
-            Vector3 originAngles = inertialAxisRotation.eulerAngles;
-            Origin inertialOrigin = new Origin(robotLink.centerOfMass.Unity2Ros().ToRoundedDoubleArray(), new double[] { (double)originAngles.x, (double)originAngles.y, (double)originAngles.z });
-            Link.Inertial.Inertia inertia = ExportInertiaData();
+            return ToLinkInertial(robotLink);
+        }
+
+        Link.Inertial ToLinkInertial(ArticulationBody robotLink)
+        {
+            var originAngles = inertialAxisRotation.eulerAngles;
+            var inertialOrigin = new Origin(
+                robotLink.centerOfMass.Unity2Ros().ToRoundedDoubleArray(), 
+                new double[] { (double)originAngles.x, (double)originAngles.y, (double)originAngles.z });
+            var inertia = ExportInertiaData();
```

Comment:
Shouldn't this be "ExportInertialData()" ? But wouldn't that cause a circular call loop? Is there another method called ExportInertiaData()? If so, naming is NOT very great x)

Reply:
Yeah as far as I can tell there are at least three different classes all called very similar things. There's the `UrdfInertial`, `Link.Inertial`, and `Link.Inertial.Inertia`... I agree it is not great but am not familiar enough with this codebase to be able to confidently suggest new names.